demos/shadow/shadow_demo_main/CMakeLists.txt

```diff
@@ -12,14 +12,8 @@ include( ${CMAKE_SOURCE_DIR}/libraries/aws/device-shadow-for-aws-iot-embedded-sd
 include( ${CMAKE_SOURCE_DIR}/libraries/standard/coreJSON/jsonFilePaths.cmake )
 
 # Demo target.
-add_executable(${DEMO_NAME})
-
-# Add to default target if all required macros needed to run this demo are defined
-check_aws_credentials(${DEMO_NAME})
-
-target_sources(
+add_executable(
     ${DEMO_NAME}
```

Comment:
Do you want to update the indentation now that "PRIVATE" is gone?

Reply:
I do like having it indented though coz it kinda shows the heirarchy that these sources are under the executable.